resources/source-types/sap_netweaver.yaml

```
@@ -0,0 +1,262 @@
+apiVersion: bindplane.observiq.com/v1
+kind: SourceType
+metadata:
+  name: netweaver
+  displayName: SAP Netweaver
+  icon: /icons/sources/sap.svg
+  description: Collect metrics from SAP Netweaver.
+spec:
+  version: 0.0.1
+  supported_platforms:
+    - linux
+  parameters:
+    # Metrics
+    - name: enable_metrics
+      label: Enable Metrics
+      type: bool
+      default: true
+
+    - name: hostname
+      label: Hostname
+      description: The hostname or IP address of the SAP Netweaver system.
+      type: string
+      default: localhost
+      required: true
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+
+    - name: port
+      label: Port
+      description: The TCP port of the SAP Netweaver system.
+      type: int
+      default: 50013
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+
+    - name: collection_interval
+      label: Collection Interval
+      description: How often (seconds) to scrape for metrics.
+      type: int
+      default: 60
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+      advancedConfig: true
+
+    - name: username
+      label: Username
+      description: The username to use when connecting to SAP Netweaver.
+      type: string
+      required: true
+      default: ""
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+      advancedConfig: true
+
+    - name: password
+      label: Password
+      description: The password to use when connecting to SAP Netweaver.
+      type: string
+      required: true
+      default: ""
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+      advancedConfig: true
+
+    - name: profile
+      label: Profile
+      description: The profile path in the form of /sapmnt/SID/profile/SID_INSTANCE_HOSTNAME to collect sapnetweaver.abap.rfc.count and sapnetweaver.abap.session.count metrics.
+      type: string
+      default: ""
+      relevantIf:
+        - name: enable_metrics
+          operator: equals
+          value: true
+      options:
+        sectionHeader: true
+      advancedConfig: true
```

If profile is optional, I think we should mention what it enables when used, and possibly link to external documentation if you think that would be useful.

```suggestion
      advancedConfig: true
      documentation:
        - text: Read more
          url: https://linktodoc.com/somepath
```

---

add docs. It is a little confusing because the [reference docs](https://help.sap.com/docs/SAP_NETWEAVER_750/b17bd20044a44edb9aa7830ee52ffac3/00fabccaa98348f291f3c7f02098116b.html) show the base path to be `/usr/sap/<SID>/SYS/profile` however when investigating and talking to SAP experts, we used `/sapmnt/SID/profile` which [are basically the same thing](https://answers.sap.com/answers/5487635/view.html) and is softlink there